### DISCUSSION TITLE
Exciting thing's using Spring boot

### DISCUSSION THREAD

**Discussion by luxmikant** [community/community#128912] [Programming Help]
### Select Topic Area

General

### Body

can anyone suggest open source projects for Spring Boot and open to working with you good people out there

**Comment by ebndev** [comment]
Hey @luxmikant, thanks for being a part of the GitHub Community!

It looks like Spring Boot's [Community page](https://spring.io/community) has a lot of resources for projects and ways to learn more about what's going on with them. Maybe a good place to start would be my opening issues for Spring projects as they mention you can `read the code for most Spring projects, raise issues, suggest changes, and view milestones and releases.`

Best of luck!
